Examples/ScalarField/InitialScalarData.hpp

```diff
@@ -0,0 +1,60 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifndef INITIALSCALARDATA_HPP_
+#define INITIALSCALARDATA_HPP_
+
+#include "Cell.hpp"
+#include "Coordinates.hpp"
+#include "MatterCCZ4.hpp"
+#include "ScalarField.hpp"
+#include "Tensor.hpp"
+#include "UserVariables.hpp" //This files needs NUM_VARS - total no. components
+#include "VarsTools.hpp"
+#include "simd.hpp"
+
+//! Class which sets the initial scalar field matter config
+class InitialScalarData
```

Comment:
Yeah, I stopped making descriptive names for initial data classes because they are always so specific. I think having a generic InitialScalarData class which you hack each time is more practical. In my code I usually just have InitialData(). It also means one less thing to change when copying an example.

Good call re iota :-) I knew there must be a function for it but couldn't find one.